### PR TITLE
#33 create social impact beyond business section

### LINKED ISSUE #33: 🎨 Create Social Impact beyond business section
#  🎨 Create Social Impact beyond business section
---

#### Note: This issue is dependent on #31, please complete that issue first

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a section that describes the organisation beyond business.

### Description

This component displays provided text and images side by side on the social impact section.

Then section has the following :

```
1. An image on the left side
2. A paragraph of text
3. A title

```

Screenshot 2023-02-06 at 10 24 53

Your assignment is to implement the component to appear as above, reuse the component on #31.

Please take note of all the design features (including mobile and rounded corners)

#### Link to full design:
```
https://xd.adobe.com/view/7b72572b-b40a-4618-bfa6-5500ab28f13b-2e21/screen/31438bb0-400a-4da8-a374-c0440535c37c

mobile 
-

#### Image link

https://res.cloudinary.com/dyl3rncv3/image/upload/v1675690301/elewa-group-website/Images/IMG_0764_twyo3k.png
```
### Technical Analysis

This component will be created in libs/pages/elewa/social-impact

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the main style.scss
4. The code is isolated to the libs/page/home package, unless a justified reason is specified.
5. The component is aligned using CSS flex
6. The design is responsive for screen, tablet and phone
7. The image renders fast and has multiple srcsets

### PR DESCRIPTION
**Pull request by Bel-94**
This pull request is part of the work on issue #33  done in collaboration with.

To achieve this, we needed to:

```
- to re-use the banner component #31
- import the BannerModule so that we can re-use the component
- Make it pixel perfect and mobile friendly
```

Fixes #33  

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Screenshot
Social impact beyond business with image on the left
Screenshot from 2023-02-17 12-06-28

Social impact beyond business with image on the left mobile version
Screenshot from 2023-02-17 11-39-24

Screenshot from 2023-02-17 11-40-03

Screenshot from 2023-02-17 11-40-22

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Bel-94** [comment]
Hi @IanOdhiambo9 kindly review my PR #219

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html [hunk 1]
```diff
@@ -1,3 +1,13 @@
+<<<<<<< HEAD
+<div class="home-page">
+  <elewa-group-header></elewa-group-header>
+
+  <elewa-group-home-hero-section></elewa-group-home-hero-section>
+  
+</div>
+
+<!-- <app-elewa-group-card></app-elewa-group-card> -->
+=======
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:10]: remove this

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html [hunk 2]
```diff
@@ -6,4 +16,5 @@
       <elewa-group-home-jobs-section></elewa-group-home-jobs-section>
     </div>
   </elewa-group-elewa-group-main-page> 
-</div>
\ No newline at end of file
+</div>
+>>>>>>> 1f5124ccd6a27958b4d7e74ec6e9dd4b5b85ce42
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:20]: remove

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html [hunk 3]
```diff
@@ -1,3 +1,13 @@
+<<<<<<< HEAD
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:1]: remove

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html [hunk 4]
```diff
@@ -1,3 +1,13 @@
...
+<div class="home-page">
```
Comment (LemmyMwaura) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:2]: This should not be in a pages folder as it's not a page.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync with the latest updates as the social impact lib was already created and work from there

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
This is too many files changed for this issue.
Please sync fork and resolve conflicts then reopen PR after changed implemented

**Pull request closed without merging**